### ISSUE TITLE
Validar estrutura de metadados

### ISSUE THREAD

**Issue by juliana-gurgel** [CompLin/nheengatu#494]
@leoalenc , na anotação das lendas de Casasnovas (2006), comecei a adotar a seguinte estrutura de metadados como base:

# sent_id = Casasnovas2006:11:18:149
# text = Amuramé, paá, uyaxiú yumasí irumu.
# text_eng = TODO
# text_por = As vezes chorava porque estava com fome.
# text_source = p. 95, No. 25
# text_orig = Amuramé, paá, uyaxiú yumasí irumu.
# text_sec = Amuramé paá uyaxiú yumasí irumu.
# text_por_sec = Às vezes ele chorava de fome.
# text_sec_source = Avila (2021)
# text_por_sec_source = Avila (2021)
# text_annotator = Juliana Lopes Gurgel
# title = Kurasí resewara
# title_orig = Kurasí Resewara
# title_por_orig = História do Sol
# title_eng = Story of the Sun
# aknowledgement = DACILAT Project, FAPESP's Process No. 2022/09158-5

Passei a utilizar essa estrutura a partir da sentença Casasnovas2006:11:16:147. Posso seguir assim?

**Comment by leoalenc** [comment]
> @leoalenc , na anotação das lendas de Casasnovas (2006), comecei a adotar a seguinte estrutura de metadados como base:
> 
> > sent_id = Casasnovas2006:11:18:149
> > text = Amuramé, paá, uyaxiú yumasí irumu.
> > text_eng = TODO
> > text_por = As vezes chorava porque estava com fome.

@juliana-gurgel , falta mesmo a crase aqui? Esta página falta no PDF.

> > text_source = p. 95, No. 25
> > text_orig = Amuramé, paá, uyaxiú yumasí irumu.
> > text_sec = Amuramé paá uyaxiú yumasí irumu.
> > text_por_sec = Às vezes ele chorava de fome.
> > text_sec_source = Avila (2021)
> > text_por_sec_source = Avila (2021)
> > text_annotator = Juliana Lopes Gurgel
> > title = Kurasí resewara
> > title_orig = Kurasí Resewara
> > title_por_orig = História do Sol

@juliana-gurgel , de modo geral, só faz sentido o sufixo `orig` se divergimos no atributo respectivo. Por exemplo, em `Casasnovas2006:1:1:1`, ` title` difere do que está em Casasnovas (2006, p. 64), pelos critérios de normalização que temos adotado:

\# title = Urubú, wirawasú
\# title_orig = Urubú, Wirá-Wasú


Via de regra, quando divergimos de algo citado, reproduzimos o original em atributo com sufixo `orig`, em se tratando de fonte primária, como neste caso.

> > title_eng = Story of the Sun
> > aknowledgement = DACILAT Project, FAPESP's Process No. 2022/09158-5
> 
> Passei a utilizar essa estrutura a partir da sentença Casasnovas2006:11:16:147. Posso seguir assim?

@juliana-gurgel , veja minhas respostas acima. A rigor, só precisamos informar metadados de título na primeira sentença de cada lenda. 
Deve haver muitas incoerências a serem corrigidas no _treebank._ Se encontrar alguma coisa estranha, abra _issue_ neste repositório.

**Comment by leoalenc** [comment]
@juliana-gurgel , pode deixar a repetição dos metadados de título se já tiver feito isso. Apenas é uma redundância que podemos evitar para ter menos trabalho. Um script pode inserir esses dados, por exemplo.

**Comment by juliana-gurgel** [comment]
> @juliana-gurgel , falta mesmo a crase aqui? Esta página falta no PDF.

Sim. Segue a foto da página. Assim que possível, vou escanear o meu livro para termos a obra completa em PDF.

![image](https://github.com/user-attachments/assets/d7c0aaf5-16a7-4a49-9fcb-fb3606525132)

**Comment by leoalenc** [comment]
> > @juliana-gurgel , falta mesmo a crase aqui? Esta página falta no PDF.
> 
> Sim. Segue a foto da página. Assim que possível, vou escanear o meu livro para termos a obra completa em PDF.
> 
> ![image](https://private-user-images.githubusercontent.com/77512676/358022667-d7c0aaf5-16a7-4a49-9fcb-fb3606525132.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM3MjkzNTcsIm5iZiI6MTcyMzcyOTA1NywicGF0aCI6Ii83NzUxMjY3Ni8zNTgwMjI2NjctZDdjMGFhZjUtMTZhNy00YTQ5LTlmY2ItZmIzNjA2NTI1MTMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE1VDEzMzczN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVkYTc3YzhhZmVmODIzOGUxOGNhYzUzODFhMGRiYzA3NDgxNjZjZGYwOWJiZjhkYjA1OWZlMTkyOTE0MDlmOTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.M6p5YNQ5uqsn5Jug0yYIiBGqmR4PeaIyuWNixKBvdmk)

@juliana-gurgel , então `text_por_orig` reproduz o teor literal do n° 25 dessa página, ao passo que  `text_por`  introduz a versão corrigida, ou seja,com crase. Via de regra, devemos adotar uma visão bem liberal (e fiel) aos textos originais, incluindo o atributo  `text_por_orig` apenas no caso de erros ortográficos óbvios. Só pra confirmar, você pode postar aqui os metadados atualizados dessa sentença bem como de 23 e 24?

**Comment by juliana-gurgel** [comment]
p. 95, No. 23-24
> sent_id = Casasnovas2006:11:17:148
> text = Nhaã pukusá, paá, Teyú raíra mirĩ, teité, arupí upunhẽ-punhẽ ukara rupí, i manha usú pukusá kupixá kití.
> text_eng = TODO
> text_por = Enquanto o filhinho do Calango, coitado, ficava engatinhando pelo terreiro quando sua mãe ia na roça.
> text_por_orig = Enquanto o filhinho do Calango, coitado, ficava engatilhando pelo terreiro quando sua mãe ia na roça.
> text_source = p. 95, No. 23-24
> text_orig = Nhaã pukusá, paá, Teyú raíra mirí, teité, arupí upunhépunhé ukara rupí, imanha usú pukusá kupixá kití.
> text_annotator = Juliana Lopes Gurgel
> aknowledgement = DACILAT Project, FAPESP's Process No. 2022/09158-5

p. 95, No. 25
> sent_id = Casasnovas2006:11:18:149
> text = Amuramé, paá, uyaxiú yumasí irumu.
> text_eng = TODO
> text_por = Às vezes chorava porque estava com fome.
> text_por_orig = As vezes chorava porque estava com fome.
> text_source = p. 95, No. 25
> text_orig = Amuramé, paá, uyaxiú yumasí irumu.
> text_sec = Amuramé paá uyaxiú yumasí irumu.
> text_por_sec = Às vezes ele chorava de fome.
> text_sec_source = Avila (2021)
> text_por_sec_source = Avila (2021)
> text_annotator = Juliana Lopes Gurgel
> aknowledgement = DACILAT Project, FAPESP's Process No. 2022/09158-5

> @juliana-gurgel , veja minhas respostas acima. A rigor, só precisamos informar metadados de título na primeira sentença de cada lenda.

Obrigada, professor. Da lenda 11 em diante, vou registrar os metadados de título dessa maneira que o senhor orienta. Os metadados das lendas anteriores deixo para corrigirmos automaticamente, como o senhor sugeriu.

**Comment by leoalenc** [comment]
Relaciona-se com #443.

**Comment by leoalenc** [comment]
@juliana-gurgel , substituí __orig_sec_ por __sec_ nos atributos das histórias de Casasnovas (2006), pois um texto não pode ser secundário e original ao mesmo tempo.
